figures/metrics.py

```diff
@@ -423,6 +432,40 @@ def calc_from_course_daily_metrics():
     return calc_from_course_daily_metrics()
 
 
+def get_total_active_courses_for_time_period(site, start_date, end_date):
+    """
+    This metric is not currently captured in SiteDailyMetrics, so retrieving from
+    CourseOverview instead
+    """
+
+    def calc_from_courses_overview():
+        edx_organizations = organizations.models.Organization.objects.filter(edlysuborganization__lms_site=site)
+
+        if edx_organizations:
+            return CourseOverview.objects.filter(
```

Comment:
Could you please add a comment here to explain a bit about this logic?

Reply:
The function description is updated.